dspace/config/emails/request_item.admin

```diff
@@ -13,6 +13,6 @@
 ${params[3]}, with address ${params[4]},
 requested the following document/file to be in Open Access:
 
-Document Handle:${params[1]}
+Document Handle: ${params[1]}
 File ID: ${params[0]}
-Token:${params[2]}
+Token: ${params[2]}
```

Comment:
The message is unsigned.  In this case the instance is not identified.  Signature should be sufficient.